portfolio/src/main/webapp/script.js

```diff
@@ -212,16 +196,41 @@ function getMessageFromJSON(pageReloadBoolean) {
 }
 
 /**
- * Creates an <li> element containing {@code text} and appends it to 
- * {@code ulElement}.
+ * Creates an <li> element containing {@code comment}'s text, timestamp,
+ *  and submitter's name and appends it to {@code ulElement}.
  */
-function appendTextToList(text, ulElement) {
+function appendTextToList(comment, ulElement) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+
+  const infoDivElement = document.createElement(DIV_TAG);
+  infoDivElement.className = INFO_CLASS;
+  
+  const date = (new Date(comment.timestamp)).toString()
+      .substring(0, END_OF_TIMESTAMP);
+
+  appendPTagToContainer(comment.name, infoDivElement);
+  appendPTagToContainer(date, infoDivElement);
+
+  liElement.appendChild(infoDivElement);
+  const textPElement = appendPTagToContainer(comment.text, liElement);
+  textPElement.className = COMMENT_CLASS;
 
   ulElement.appendChild(liElement);
 }
 
+/**
+ * Creates a p tag to store the given {@code text} inside the
```

Comment:
```suggestion
 * Creates a <p> tag to store the given {@code text} inside the
```

Reply:
You can rephrase this as: Creates and returns a <p> tag. That way you can remove the @return section.

Reply:
Done!